daemon/src/engine/client/cl_main.cpp

```diff
@@ -50,6 +50,8 @@ cvar_t *cl_wavefilerecord;
 #include "framework/Crypto.h"
 #include "framework/Network.h"
 
+#include "../common/Profiler.h"
```

Comment:
I think common/Profiler.h should work. Also headers in the common/ folder should be put in common/Common.h so that precompiled headers pick them up.